modules/management-console-agent/src/test/resources/ignite-cfg.xml

```diff
@@ -41,6 +41,14 @@
             </bean>
         </property>
 
+        <property name="cacheConfiguration">
```

Comment:
Why not simply use `ignite.getOrCreateCache("test-cache")` ?

Reply:
Because we use this configuration from CommandLineLauncher and we can't use getOrCreate :(